src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -126,19 +126,147 @@ class PerformanceLattice(BaseModel):
     Attributes:
         positionErrorAbs (Decimal): Error between target and actual site
```

Comment:
We also want to modify description here.

Reply:
Thanks for catching it. Addressed